lib/gui/models/drives.js

```diff
@@ -20,103 +20,94 @@
  * @module Etcher.Models.Drives
```

Comment:
Not sure I understand - would you mind clarifying that a bit for this JS newbie? :wink: